R/credentials.R

```diff
@@ -0,0 +1,32 @@
+#' Get username and password
+#'
+#' Defaults to get from environment variable, falling back to getting them
+#' interactively if supported
+#'
+#' @return The username and password entered
+#' @keywords internal
+get_credentials <- function() {
+  list(
+    username = get_single_credential("SHAREPOINT_USERNAME",
```

Comment:
see https://github.com/vimc/montagu-r/blob/3e9359af079f7eee08e5b60a240fcc4db8d9e09e/R/auth.R#L204-L214

Reply:
Thanks - tidied that up